app/templates/auth/update-user.html

```diff
@@ -0,0 +1,45 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Create contributor account – Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+{% if not token %}
+
+<header class="page-heading-smaller">
+  <h1>This link is invalid</h1>
+</header>
+<p>
+    Check you&#146;ve entered the correct link or ask the person who invited you to send a new invitation.
```

Comment:
You can use UTF-8 characters in Jinja2 so I think either this should be a UTF-8 character or the apostrophe on https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/131/files#diff-fd6ab505bb4711c18d7b63bd49b6e2e7R14 should be changed.